test_unstructured_ingest/evaluation-metrics.sh

```diff
@@ -14,31 +14,54 @@ EVAL_NAME="$1"
 # Download cct test from s3
 BUCKET_NAME=utic-dev-tech-fixtures
 FOLDER_NAME=small-eval-"$EVAL_NAME"
-LOCAL_EVAL_SOURCE_DIR=$SCRIPT_DIR/gold-standard/$FOLDER_NAME
-mkdir -p "$LOCAL_EVAL_SOURCE_DIR"
-aws s3 cp "s3://$BUCKET_NAME/$FOLDER_NAME" "$LOCAL_EVAL_SOURCE_DIR" --recursive --no-sign-request --region us-east-2
+SOURCE_DIR=$SCRIPT_DIR/gold-standard/$FOLDER_NAME
+mkdir -p "$SOURCE_DIR"
+aws s3 cp "s3://$BUCKET_NAME/$FOLDER_NAME" "$SOURCE_DIR" --recursive --no-sign-request --region us-east-2
```

Comment:
to confirm, is there a cleanup of everything out of the source dir at the end of this process? if not, the aws cp command will start to build up duplicates, and you may want to consider using `aws s3 sync` instead to reflect updated/deleted files.